modules/join/src/main/java/org/apache/ignite/internal/join/InitException.java

```diff
@@ -15,8 +15,17 @@
  * limitations under the License.
  */
 
+package org.apache.ignite.internal.join;
+
 /**
- * Configuration schemas for Cluster node.
+ * Exception thrown when cluster initialization fails for some reason.
  */
+public class InitException extends RuntimeException {
```

Comment:
It seems to me, this exception should extend `IgniteInternalException` instead of `RuntimeException`